src/measure_theory/function/strongly_measurable.lean

```diff
@@ -0,0 +1,345 @@
+/-
+Copyright (c) 2021 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+
+import measure_theory.function.simple_func_dense
+
+/-!
+# Strongly measurable and finitely strongly measurable functions
+
+A function `f` is said to be strongly measurable with respect to a measure `μ` if `f` is the
+sequential limit of simple functions. It is said to be finitely strongly measurable if the supports
+of those simple functions have finite measure.
```

Comment:
the measure is only relevant for the second definition. Can you move its mention to the second sentence?